response_operations_ui/templates/conversation-view/cv-reply-form.html

```diff
@@ -0,0 +1,26 @@
+<div class="secure-message-form">
+    <form action="{{ url_for('messages_bp.view_conversation', thread_id=messages[0].thread_id, page=page) }}" method="post" id="create-message-form">
+        {{ form.csrf_token }}
+        {{ form.subject(id='secure-message-subject', type='hidden', value=subject) }}
+        {{ form.hidden_to_ru_id(type='hidden', value=ru_id) }}
+        {{ form.hidden_survey_id(type='hidden', value=survey) }}
+        {{ form.hidden_to_uuid(type='hidden', value=to_uuid)}}
```

Comment:
kinda unrelated, but I hope to god we're not using anything like this in frontstage secure messaging

Reply:
I don't know, to be honest.  Do you mean for security reasons?

Reply:
yes